sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/keyvault_pipeline.hpp

```diff
@@ -1,11 +1,19 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+/**
+ * @brief Provides a wrapper class for the Azure Core Pipeline for all Key Vault services where
```

Comment:
Putting the declaration at the beginning of the line makes me wonder whether new lines are significant here, since line 6 is a new line. If not, please ignore.